group-actors/src/main/java/org/sunbird/service/GroupServiceImpl.java

```diff
@@ -96,7 +97,8 @@ public void readGroupActivities(Map<String, Object> dbResGroup) {
    *
    * @param dbResActivities
    */
-  private void addActivityInfoDetails(List<Map<String, Object>> dbResActivities) {
+  private void addActivityInfoDetails(
+      List<Map<String, Object>> dbResActivities, Map<String, Object> reqContext) {
     logger.info("Fetching activityInfo for activity count: {}", dbResActivities.size());
     Map<SearchServiceUtil, Map<String, String>> idClassTypeMap =
         GroupUtil.groupActivityIdsBySearchUtilClass(dbResActivities);
```

Comment:
context

Reply:
We use MDC to log traceId in groups, so there is no requirement to pass request context in each method to log traceId